### DISCUSSION TITLE
Disable Security onion Firewall?

### DISCUSSION THREAD

**Discussion by lmb2100** [Security-Onion-Solutions/securityonion#10414] [General]
Hello all, is there a way to disable the firewall? I assume you would have to do this on each manager and node but I am troubleshooting an issue and would like to see if I could disable the firewall to see if that's the issue, (obviously not recommended but this is not connected to internet) thanks!

**Comment by TotieBash** [comment]
systemctl stop firewalld
systemctl start firewalld

**Comment by TOoSmOotH** [comment]
disabling the firewall will break all kinds of stuff. tcpdump should give you all the information you need.
